src/redux/reducers/index.js

```diff
@@ -0,0 +1 @@
+export { default as todos } from './todoReducer';
```

Comment:
Unnecessary index file. It's better to export the return value of `combineReducers` directly